slidez-extension/src/components/poll/Poll.tsx

```diff
@@ -34,7 +41,7 @@ function Poll({ poll, children }: PollProps) {
                         style={{ width: percentageFormat }}
                     />
                     <div className='poll-option-percent'>
-                        {percentageFormat}
+                        {animatePercent}
```

Comment:
Please, execute functions only on some events, not like a code part (You can make so only if it returns some data)